zstd-safe/zstd-sys/build.rs

```diff
@@ -203,7 +199,9 @@ fn main() {
     }
 
     // println!("cargo:rustc-link-lib=zstd");
-    let (defs, headerpaths) = if cfg!(feature = "pkg-config") {
+    let (defs, headerpaths) = if cfg!(feature = "pkg-config")
```

Comment:
We may need to add `pkg-config = []` to the features array in `Cargo.toml` to allow setting this as a feature.